integration/msi/tools/msiversion/msiversionconverter.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+func main() {
```

Comment:
Would be nice if you can put some comment here explaining why we have to do the conversion, something along the line of the internal integ-testing repo: 
        ``` msi version is different from the agent original version because of the msi limit
        Product version must have a major version less than 256, a minor version less than 256, and a build version less than 65536.
        Based on https://w.amazon.com/index.php?title=BrazilBuildSystem/Concepts/PackageVersion&oldid=49363397, the z version is no
        longer used since January 5th 2018 ```